assayist/processor/container_go_analyzer.py

```diff
@@ -105,16 +119,46 @@ def run(self):
             raise AnalysisFailure('GoAnalyzer completed with the following error(s): \n  {}'
                                   .format("\n  ".join(failures)))
 
-    def _process_source_code(self, source_location, srcdir, excludes=None):
+    def _get_import_path_override(self, srcdir):
+        # Look inside the Dockerfile for a named label
+        label = 'io.openshift.source-repo-url'
+        try:
+            df = DockerfileParser(srcdir, cache_content=True)
+        except IOError:
+            log.exception("Unable to read Dockerfile")
```

Comment:
Could you please use single quotes since the rest of the project uses them?

Reply:
Fixed.